### PR TITLE
added 4_modeling_keras solution, data csv files

### PR DESCRIPTION
**Pull request by munnm**

### CODE REVIEW

File: courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb [hunk 1]
```diff
@@ -0,0 +1,1261 @@
+{
```
Comment (crawles) [inline courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>do you need to map the "objectives" to individual parts of the code as we did for other labs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (crawles) [inline courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>import datetime</p><p>import shutil</p><p> </p><p>these are unnecessary, see comments below.</p><p> </p><p>can you please add the following lines?</p><p> </p><p># For reproducible results.</p><p>from numpy.random import seed</p><p>seed(1)</p><p>from tensorflow import set_random_seed</p><p>set_random_seed(2)</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (crawles) [inline courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>just use</p><p> </p><p>%env PROJECT=PROJECT</p><p>%env ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (crawles) [inline courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>"Lstly" grammer error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (crawles) [inline courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>love these visualizations!!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (crawles) [inline courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>... or add any other addition ... ---&gt; "additional". fix spelling of techniques</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (crawles) [inline courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>Capitalize TensorFlow, remove extra period a&nbsp;<a href="https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv1D" target="_blank">Conv1d in tensorflow.</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (crawles) [inline courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>instead of shutil, just do ! rm -r $LOGDIR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Comment (crawles) [inline courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>We are not using the features that we made in my lab for feature engineering. </p><p> </p><p>close_MIN_prior_5_daysclose_MIN_prior_20_daysclose_MIN_prior_260_daysclose_MAX_prior_5_daysclose_MAX_prior_20_daysclose_MAX_prior_260_daysclose_AVG_prior_5_daysclose_AVG_prior_20_daysclose_AVG_prior_260_daysclose_STDDEV_prior_5_daysclose_STDDEV_prior_20_daysclose_STDDEV_prior_260_days</p><p> </p><p>can you make a bonus that says for them to try incorporating these features and see if it boosts model performance?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>Done. </p><p>Note, <code>tf.set_random_seed(seed)</code>&nbsp;has changed to&nbsp;<code>tf.random.set_seed(seed)</code>&nbsp;in TensorFlow 2.</p>

Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>done</p>

Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>:)</p>

[the remark this answers is not in the document]
Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>Yes. Using eps_percent_change_sp500 now</p>

Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>decided to get rid of this section since it's less relevant with the new dataset and they already have examples of similar input functions. </p>

Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>done</p>

[the remark this answers is not in the document]
Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>done</p>

Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>done</p>

Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>done</p>

Comment (munnm) [reply courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb]: <p>done</p>

**Comment by munnm** [comment]
<p>thanks Chris, have incorporated your comments</p>

**Comment by crawles** [comment]
<p>Overall looks great. </p><p> </p><p>1) Lab is out of date, can you please update?</p><p>2) I don't think we need the "large" csv files as we are not using them anymore, right?</p><p> </p><p>After you address these I'll merge</p>

**Comment by munnm** [comment]
<p>ok. Updated the lab, made TODOs, and moved to 'labs' folder. Also, removed large csv files. Thanks!</p>
